server/models/User.coffee

```diff
@@ -652,15 +652,15 @@ UserSchema.statics.makeNew = (req) ->
       userGeo = {}
       userGeo.country = geo.country
       if country = geo.country
-          userGeo.countryName = countryList.getName(country)
+          userGeo.countryName = _.str.slugify(countryList.getName(country))
       userGeo.region = geo.region
       userGeo.city = geo.city
       userGeo.ll = geo.ll
       userGeo.metro = geo.metro
       userGeo.zip = geo.zip
       user.set 'geo', userGeo
   if req.country                        # Storing the country name again since user.country is being used in various other files
-    user.set 'country', req.country
+    user.set 'country', _.str.slugify(req.country)
   else if userGeo?.countryName
     user.set 'country', userGeo.countryName
```

Comment:
Will this overwrite an existing user.country with a different value?  I thought we were going to leave user.country as-is if these two values were different.

Reply:
This is when creating a new user - so when req.country exists we set that in the user.country(as it was before doing  the geo changes), if not then its set to geo.countryName. Here, I just changed it to be a slugified value.
For the existing users, I am leaving user.country as-is if both values are different ( lines 24-33 in the script above)

Reply:
👍